app/controllers/users_controller.rb

```diff
@@ -18,9 +18,25 @@ def show
     require_user
   end
 
+  def edit
+    @user = User.find(session[:user])
+  end
+
+  def update
```

Comment:
update is a really long method now, we should split it off into private helper methods

Reply:
Added to the to do list